<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Face_Detection</class>
 <widget class="QWidget" name="Face_Detection">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>471</width>
    <height>372</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Face_Detection</string>
  </property>
  <widget class="QWidget" name="verticalLayoutWidget">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>230</y>
     <width>160</width>
     <height>100</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="QLineEdit" name="le_name">
      <property name="placeholderText">
       <string>نام:</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLineEdit" name="le_family">
      <property name="placeholderText">
       <string>نام خانوادگی:</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLineEdit" name="le_brith">
      <property name="placeholderText">
       <string>تاریخ تولد:</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLineEdit" name="le_natinalcode">
      <property name="placeholderText">
       <string>کد ملی:</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="horizontalLayoutWidget">
   <property name="geometry">
    <rect>
     <x>190</x>
     <y>240</y>
     <width>64</width>
     <height>61</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout">
    <item>
     <widget class="QPushButton" name="Cam_b">
      <property name="font">
       <font>
        <family>Roboto Condensed Light</family>
        <pointsize>10</pointsize>
       </font>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="icon">
       <iconset>
        <activeon>../بارگیری.png</activeon>
       </iconset>
      </property>
      <property name="iconSize">
       <size>
        <width>50</width>
        <height>50</height>
       </size>
      </property>
      <property name="checkable">
       <bool>true</bool>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="verticalLayoutWidget_2">
   <property name="geometry">
    <rect>
     <x>120</x>
     <y>10</y>
     <width>202</width>
     <height>202</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout_2">
    <item>
     <widget class="QLabel" name="img_label">
      <property name="minimumSize">
       <size>
        <width>200</width>
        <height>200</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>200</width>
        <height>200</height>
       </size>
      </property>
      <property name="styleSheet">
       <string notr="true">
background-color: rgb(255, 255, 255);</string>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="verticalLayoutWidget_3">
   <property name="geometry">
    <rect>
     <x>260</x>
     <y>230</y>
     <width>160</width>
     <height>80</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout_3">
    <item>
     <widget class="QPushButton" name="pb_add">
      <property name="text">
       <string>اضافه کردن کارمند جدید</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
